README.md

```diff
@@ -0,0 +1,17 @@
+# browserify-rsa
```

Comment:
eh since license is all caps as well, we should probably make this one all caps as well